content/contribute/terms-and-conditions.md

```diff
@@ -7,3 +7,43 @@ linked:
 ---
 
 ## HCA Data Contributor Terms and Conditions
+
+### Preliminary terms and conditions for data use and sharing
+
+Version: 
+
+As soon as the official Human Cell Atlas terms and conditions for data contribution are finalised and reviewed by our legal authorities, you will be asked to agree to them. In that instance, we will delete all data you have already submitted to the Human Cell Atlas Data Coordination Platform.
```

Comment:
> In that instance, we will delete all data you have already submitted to the Human Cell Atlas Data Coordination Platform.

Isn't this only if they then say no? We aren't going to delete data if they agree to the new terms.